tensorflow_datasets/image_classification/kaokore/dummy_data_generation.py

```diff
@@ -13,22 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+r"""Generate kaokore data.
+
+"""
+
 import os
-import random
 
 from absl import app
 from absl import flags
 
 import tensorflow.compat.v2 as tf
 from tensorflow_datasets.core.utils import py_utils
```

Comment:
There are a couple more linting errors, which might have gone unnoticed in the prior versions of `pylint`. Could you update `pylint` to the latest version and give it a final go? Else it looks good!

Reply:
Hi @NikhilBartwal . I am using `pylint 2.6.0` and this [pylintrc](https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/pylintrc) file, it gave me a rating a rating of `10.00/10`. Here my environments and results

```sh
$ pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.9.0 (default, Oct 27 2020, 14:13:35)
[Clang 11.0.0 (clang-1100.0.33.17)]
$ shasum pylintrc
34d829b669fb1121a9ad32587e61e496cf44eca2  pylintrc
$ git rev-parse --short HEAD
b9c0ffd5
$ pylint --rcfile=pylintrc tensorflow_datasets/image_classification/kaokore

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

Am I missing anything?

Reply:
@stwind Are there some changes that you have not pushed yet? It generates the following on my system with the same environment:

```
C:\Users\nikhil>pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.9.1 (tags/v3.9.1:1e5d33e, Dec  7 2020, 17:08:21) [MSC v.1927 64 bit (AMD64)]

C:\Users\nikhil>pylint --rcfile=pylintrc F:/tfds/datasets/tensorflow_datasets/image_classification/kaokore
************* Module tensorflow_datasets.image_classification.kaokore.dummy_data_generation
F:\tfds\datasets\tensorflow_datasets\image_classification\kaokore\dummy_data_generation.py:26:0: C0411: third party import "from tensorflow_datasets.core.utils import py_utils" should be placed before "import tensorflow.compat.v2 as tf" (wrong-import-order)
F:\tfds\datasets\tensorflow_datasets\image_classification\kaokore\dummy_data_generation.py:27:0: C0411: third party import "from tensorflow_datasets.testing import fake_data_utils" should be placed before "import tensorflow.compat.v2 as tf" (wrong-import-order)
************* Module tensorflow_datasets.image_classification.kaokore.kaokore
F:\tfds\datasets\tensorflow_datasets\image_classification\kaokore\kaokore.py:5:0: C0411: third party import "import tensorflow_datasets.public_api as tfds" should be placed before "import tensorflow.compat.v2 as tf" (wrong-import-order)

------------------------------------------------------------------
Your code has been rated at 9.56/10 (previous run: 9.56/10, +0.00)
```

Reply:
I just manually reordered the imports according to your linting results, can you take another look?